### PR TITLE
docs: Update index.md

### PR DESCRIPTION
**Pull request by syedjawadakhtar**
# Checklist

- [ ] fixes #xxxx
- [x] Passes `Semantic Pull Request` and `TypoCheck`
- [ ] New docs are listed in `README.md`
- [x] New docs are listed in `index.md`
- [ ] Updated table of content
- [ ] Updated nav links in mkdocs.yml

---

## PR description

1. Removed redundant information about each team - which has been covered in  README of each team
2. Add the format of documentation reports
3. Update How to document 
4. Remove typos; edit image source; Update what not to document; update team names
5. Add image for index page

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -4,46 +4,67 @@
...
-1. Those which do not have concrete evidence for the result (The student can better read plenty of research papers without a concrete goal).
-2. Comparative studies - This is not journal publishing stories. Here students must benefit from the technique and not stay confused while choosing one.
+1. Those which do not have concrete evidence for the result.
+2. Redundant information that has already been done by previous members.
+
+### Format of docs
+
+The report must be in the format:
+
+```title:``` Title of the report
```
Comment (pnijhara) [inline docs/index.md]: nit pick: Single back ticks also work the same way for a single linke code block.

File: docs/index.md [hunk 2]
```diff
@@ -4,46 +4,67 @@
...
 
-1. Always start with a heading.
-2. Describe the goal.
-3. Move on to describe the steps involved for reaching the goal (From theoretical knowledge - practical steps)
-    - This must include the links
-    - The **exact** steps to be followed from the given links(include the misconceptions and the blunders that you did while doing the same - be a senpai to a newborn!!!)
-    - The output/result
-    - [OPTIONAL but IMPORTANT] If the team members had chosen a different path earlier to execute the task and it turned out to be delusional; add those techniques and results too.
-4. Finish it off with writing the name of the team member who performed it with their contact details.
+```Equipments/Software used:``` Hardware used, software with its versions
```
Comment (pnijhara) [inline docs/index.md]: These can be bold text but not code blocks.

File: docs/index.md [hunk 3]
```diff
@@ -55,57 +76,25 @@ Some other rules that everyone has to follow for uniformity in the information c
...
-## Teams
-
-1. [Computer](computer)
-
-2. [Electrical/Electronics](electronics)
-
-3. [Marketing](marketing)
-
-4. [Mechanical](mechanical)
+**NOTE:** The coordinator must make sure that the documentation can be put in the public domain with the permission of the authors. If the material provided has been submitted for publishing a research paper, the docs **must not be uploaded**, as this may ruin the opportunity of the paper to be published. The paper when published, can be linked in the docs, later.
```
Comment (pnijhara) [inline docs/index.md]: Always write note as quotation.
use ">"

**Review by pnijhara — CHANGES_REQUESTED** [review]

**Comment by pnijhara** [comment]
Just small changes required.
Rest all looks good.

**Comment by pnijhara** [comment]
These changes should go to `README` as well.
What do you think?

**Comment by syedjawadakhtar** [comment]
Yes. A link can be added in each README for the format of writing the docs. Or a link to index itself

**Review by pnijhara — APPROVED** [review]

**Pull request merged**